scripts/ort-evaluator.sh

```diff
@@ -11,16 +11,49 @@ else
     ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_SCANNER_FILE
 fi
 
+if [[ -s "$ORT_CONFIG_LICENSE_CONFIGURATION_FILE" ]]; then
+    ORT_EVALUATOR_OPTIONS="--license-classifications-file $ORT_CONFIG_LICENSE_CONFIGURATION_FILE"
+else
+    ORT_EVALUATOR_OPTIONS=""
+fi
+
+if [[ -d "$ORT_CONFIG_PACKAGE_CONFIGURATION_DIR" ]]; then
+    ORT_EVALUATOR_OPTIONS="$ORT_EVALUATOR_OPTIONS --package-configuration-dir $ORT_CONFIG_PACKAGE_CONFIGURATION_DIR"
+elif [[ -s "$ORT_CONFIG_PACKAGE_CONFIGURATION_FILE" ]]; then
+    ORT_EVALUATOR_OPTIONS="$ORT_EVALUATOR_OPTIONS --package-configuration-file $ORT_CONFIG_PACKAGE_CONFIGURATION_FILE"
+elif [[ -d "$ORT_CONFIG/packages" ]]; then
+    # Use legacy named package configuration directory if present.
+    ORT_EVALUATOR_OPTIONS="$ORT_EVALUATOR_OPTIONS --package-configuration-dir $ORT_CONFIG/packages"
+    ORT_CONFIG_PACKAGE_CONFIGURATION_DIR=$ORT_CONFIG/packages
+fi
+
+if [[ ! -s "$ORT_CONFIG_RULES_FILE" ]]; then
+    # Use legacy named policy rules file if present
+    if [[ -s "$ORT_CONFIG_DIR/rules.kts" ]]; then
+        ORT_CONFIG_RULES_FILE="$ORT_CONFIG_DIR/rules.kts"
+    else
+        # Use first found file with .rules.kts extension 
+        FIND_RESULTS=$(find $ORT_CONFIG_DIR -type f -name '*.rules.kts')
+        RESULT_FILES=( $FIND_RESULTS )
+        ORT_CONFIG_RULES_FILE="${RESULT_FILES[0]}"
+    fi
+fi
+
+if [[ -s "$ORT_CONFIG_RULES_FILE" ]]; then
+    echo "Using ORT policy rules file found in $ORT_CONFIG_RULES_FILE..."
+else
+    echo "Error: ORT policy rules file (*.rules.kts) not found. Please set ORT_CONFIG_RULES_FILE"
+fi
```

Comment:
Determining the parameters for the "config" files is a bit redundant across the respective ORT CLI invocations. 
That could be avoided by using default file locations,e.g. see [1].

I'm ok with the change as-is for now. just as an idea for the future: 

Would it make sense to refactor it for default locations?
If locations shall remain customizable maybe we can have one function which creates a default layout using symlinks to the custom locations?

[1] https://github.com/oss-review-toolkit/ort/blob/ff4ac8e3d7960fe68e4ba1640ec63fb361bc77d2/cli/src/main/kotlin/commands/EvaluatorCommand.kt#L137

Reply:
> Determining the parameters for the "config" files is a bit redundant across the respective ORT CLI invocations.

Yes, fully aware of this fact - actually have a refactoring for it on my to-do list already..